css/style.css

```diff
@@ -5,3 +5,22 @@ html {
 }
 
 /*put all other styles below this line, please*/
+#body {
```

Comment:
there won't be a `body` ID... you'll need to use the `body` element selector here!